roles/ceph-validate/tasks/main.yml

```
@@ -36,6 +36,15 @@
     - ntp_service_enabled
     - ntp_daemon_type not in ['chronyd', 'ntpd', 'timesyncd']
 
+# Since NTPd can not be installed on Atomic...
+- name: abort if ntp_daemon_type is ntp on Atomic
+  fail:
+    msg: installation can't happen on Atomic and ntpd needs to be installed
+  when:
+    - is_atomic
+    - ansible_os_family == 'RedHat'
+    - ntp_daemon_type == 'ntpd'
```

```suggestion
    - ntp_daemon_type != 'timesyncd'
```

Wouldn't this be more appropriate?

---

chrony is present on Atomic, by default. So we can allow it.

---

Then `!= 'chrony'`?

---

timesyncd comes with systemd, so that too is fine.